charts/agent-control-deployment/Chart.yaml

```diff
@@ -4,7 +4,7 @@ description: A Helm chart to install New Relic Agent Control on Kubernetes
 
 type: application
 
-version: 0.0.66
+version: 0.0.67
```

Comment:
Maybe we can avoid bumping the version because we are going to merge to a feature branch, although it will make the CI unhappy 🙃